templates/base.html

```diff
@@ -7,28 +7,13 @@
     <meta name="description" content="">
     <title>{% block title %}Imaging experiments tracking{% endblock %}</title>
 
-
     <!-- Bootstrap core CSS -->
-<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
-<meta name="theme-color" content="#563d7c">
-
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+    <meta name="theme-color" content="#563d7c">
 
-    <style>
-      .bd-placeholder-img {
-        font-size: 1.125rem;
-        text-anchor: middle;
-        -webkit-user-select: none;
-        -moz-user-select: none;
-        -ms-user-select: none;
-        user-select: none;
-      }
+    <!-- Font Awesome CSS-->
```

Comment:
this comment is misplaced and there's no need in it, I guess, since it just repeats the name of the file included. To make it more informative, you can say where it is used, like "needed for fonts in the main page", then if those fonts are not needed anymore, it will be easier to identify unnecessary css and remove it.